--- libraries/th-orphans/src/Language/Haskell/TH/Instances.hs.old	2012-11-23 05:15:32.692122407 +1300
+++ libraries/th-orphans/src/Language/Haskell/TH/Instances.hs	2012-11-23 05:15:49.595707599 +1300
@@ -11,8 +11,8 @@
   Portability :  portable (template-haskell)
 -}
 
--- | Provides Ord and Lift instances for the TH datatypes.  Also provides
---   Show / Eq for Loc, and Ppr for Loc / Lit.
+--  Provides Ord and Lift instances for the TH datatypes.  Also provides
+--  Show / Eq for Loc, and Ppr for Loc / Lit.
 module Language.Haskell.TH.Instances () where
 
 import Language.Haskell.TH.Syntax
